docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+
+One of the key benefits of .NET Core is its portability. You can package and deploy the CoreCLR with your application, eliminating your application's dependency on an installed version of the full .NET Framework at the device or server level. You can host multiple applications side-by-side using different versions of the CoreCLR, and upgrade them individually, rather than being forced to upgrade all of them simultaneously.
+
+Another benefit is common access to standard libraries. The CoreFX library includes Collections, Console access, Diagnostics, IO, LINQ, JSON, XML, and regular expression support, just to name a few. Applications targeting a variety of devices and deployment models (such as Windows Desktop, Windows Store, Windows Phone, and ASP.NET) can directly target these libraries, avoiding the need to write wrapper code or conditionally compile based on target platform, as is common today.
+
+Motivation Behind .NET Core
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The fragmentation of .NET and its capabilities across different platforms is the primary motivation for .NET Core. When .NET first shipped in 2002, it was a single framework, but it didn't take long before the .NET Compact Framework shipped, providing a smaller version of .NET designed for mobile devices. Over the years, this exercise was repeated multiple times, so that today there are different flavors of .NET specific to different platforms. Add to this the further fragmentation created by Mono and Xamarin, which target Linux, Mac, and native iOS and Android devices, and the need for a single common Core of .NET that works the same across all of these platforms becomes even more apparent. Since .NET Core is a full open source project, it will no longer be necessary for Mono to maintain its own fork of this technology, and instead Microsoft and the Mono community can collaborate together on .NET Core.
+
+.. note:: The `.NET Framework Client Profile`_ shared some of the goals of .NET Core when it was provided with .NET 4 and earlier versions, providing a subset of the .NET Framework optimized for client applications. It was later discontinued with the release of the .NET Framework 4.5.
+
+.. _`.NET Framework Client Profile`: https://msdn.microsoft.com/en-us/library/cc656912%28v=vs.110%29.aspx
+
+In addition to being able to target a variety of different device platforms, there was also pressure from the server side to reduce the overall footprint of the .NET framework. By factoring the CoreFX libraries and minimizing the size of CoreFX, server-based applications built with ASP.NET 5 can minimize their size, allowing cloud-based hosting environments to increase the density of applications they can support on a given set of hardware resources. This will provide increased economies of scale for ASP.NET applications hosted on Azure, for instance.
```

Comment:
I think that I'm trying to make a subtle point:

- .NET Core deployments can be small because its compontentized. It's pay for play.
- The overall size of .NET Core (runtime and framework) doesn't intend to be smaller than the .NET Framework over time, but pay-for-play.